cmake/InstallPioneer.cmake

```diff
@@ -31,4 +31,6 @@ if (NOT PIONEER_DATA_DIR)
 		"Runtime path to load game data from" FORCE)
 endif (NOT PIONEER_DATA_DIR)
 
-file(TO_NATIVE_PATH ${PIONEER_DATA_DIR} PIONEER_DATA_RUNTIME_DIR)
+if(NOT APPIMAGE_BUILD)
+    file(TO_NATIVE_PATH ${PIONEER_DATA_DIR} PIONEER_DATA_RUNTIME_DIR)
+endif(NOT APPIMAGE_BUILD)
```

Comment:
I think this change is wrong.

You do this so that it won't overwrite `PIONEER_DATA_RUNTIME_DIR` that you manually set on cmake's command line. It sounds to me like you could very well set `PIONEER_DATA_DIR` instead, no?

And since we're talking about paths, wouldn't it make more sense to do an in-place install with `PIONEER_INSTALL_INPLACE`? And have the data folder relative to the executable? That sounds like it would fit AppImages way better.